test/PushNotificationTests/APITests.cpp

```diff
@@ -0,0 +1,239 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#include "pch.h"
+#include <TestDef.h>
+#include "Shared.h"
+#include "MockBackgroundTaskInstance.h"
+
+using namespace WEX::Common;
+using namespace WEX::Logging;
+using namespace WEX::TestExecution;
+
+using namespace winrt::Windows::ApplicationModel::Activation;
+using namespace winrt::Windows::ApplicationModel::Background;
+using namespace winrt::Windows::Foundation;
+using namespace winrt::Windows::Foundation::Collections;
+using namespace winrt::Windows::Management::Deployment;
+using namespace winrt::Windows::Storage;
+using namespace winrt::Windows::System;
+
+namespace Test::PushNotifications
+{
+    class APITests
+    {
+    private:
+        wil::unique_event m_failed;
+
+        const std::wstring c_testPackageFile = g_deploymentDir + L"PushNotificationsTestAppPackage.msix";
```

Comment:
Should be `std::filesystem::path` rather than a simple string

```
std::filesystem::path GetTestPackageFile() const
{
    auto filename{ std::filesystem::path(g_deploymentDir) };
    filename /= L"Push...";
    return filename;
}
```